src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -284,12 +285,16 @@ class EstimateYourBenefitsForm extends React.Component {
     );
   };
 
-  renderTuition = () => {
-    if (!this.props.displayedInputs.tuition) return null;
+  renderInStateTuition = () => {
+    if (
+      !this.props.displayedInputs.tuition &&
+      this.props.inputs.inState === 'yes'
+    )
+      return null;
```

Comment:
Please add braces for clarity.

Reply:
Removed as the check is now in `ExpandingGroup` property `open` since this question is the only conditionally displayed question for its parent question